utils/download_caffe2_models.sh

```diff
@@ -26,6 +26,7 @@ vgg19
 zfnet512
 bvlc_alexnet
 en2gr
+quant_resnet50
```

Comment:
kinda unrelated question but why don't we download directly from the c2 zoo?

Reply:
Their model has a problem: for their init_net.pb/init_net.pbtxt file, it has duplicated tensors which is unexpected. I removed the duplicated tensors in our version. According to the model owner, they will update the model on their side in the next release.